figures/metrics.py

```diff
@@ -423,6 +432,42 @@ def calc_from_course_daily_metrics():
     return calc_from_course_daily_metrics()
 
 
+def get_total_active_courses_for_time_period(site, start_date, end_date):
+    """
+    Return Active courses in month based on Course Start and End Date.
+
+    This metric is not currently captured in SiteDailyMetrics, so retrieving from
+    CourseOverview instead.
+    """
+
+    def calc_from_courses_overview():
+        edx_organizations = organizations.models.Organization.objects.filter(edlysuborganization__lms_site=site)
+
+        if edx_organizations:
+            return CourseOverview.objects.filter(
+                org=edx_organizations[0].short_name
+            ).filter(
```

Comment:
According to the description we need course only between `start_date` and `end_date` 
Filtering on coursestart_date >= start_date & course_end_date<= start_date fulfils that perfectly why do we need to other filtrations? This is giving vibes we are getting all active courses here.

Reply:
There are multiple use cases against period(monthly) i.e course started before and ending during the period, course started ended during the period, etc. We have to address all use-cases.